BGAnimations/ScreenGameplay in/MeasureCounterAndModsLevel.lua

```diff
@@ -1,3 +1,67 @@
+-- Get the start/end of each stream and break sequence in our table of measures
+local GetStreamSequences = function(notesPerMeasure, notesThreshold)
+	local streamMeasures = {}
+	for i,n in ipairs(notesPerMeasure) do
+		if n >= notesThreshold then
+			table.insert(streamMeasures, i)
+		end
+	end
+
+	local streamSequences = {}
+	-- Count every measure as stream/non-stream.
+	-- We can then later choose how we want to display the information.
+	local measureSequenceThreshold = 1
+
+	local counter = 1
+	local streamEnd = nil
+
+	-- First add an initial break if it's larger than measureSequenceThreshold
+	if #streamMeasures > 0 then
+		local breakStart = 0
+		local k, v = next(streamMeasures) -- first element of a table
+		local breakEnd = streamMeasures[k] - 1
```

Comment:
Isn't `streamMeasures[k]` == `v`? Or am I misunderstanding how this works?

Reply:
You're correct. I don't actually remember my reasoning, but I think it's because below we use `streamMeasure[k+1]` for the `nextVal` so that it looked reasonable to also keep it as `streamMeasure[k]`. I changed it to the variables are more descriptive.